docs/ClusterSetup.md

```diff
@@ -221,6 +224,7 @@ zookeeperServers=zk1.us-west.example.com:2181,zk2.us-west.example.com:2181,zk3.u
 globalZookeeperServers=zk1.us-west.example.com:2184,zk2.us-west.example.com:2184,zk3.us-west.example.com:2184
 
 clusterName=us-west
+statusFilePath=broker-status-file
```

Comment:
This setting is required. The broker will fail to start with a cryptic error. The error message could be improved, it does not mention that the required setting is missing in configuration.